O365-iOS-Microsoft-Graph-Connect-Swift/AuthenticationClass.swift

```diff
@@ -50,27 +53,29 @@ class AuthenticationClass {
                                                _ accessToken: String) -> Void) {
         do {
             if let initError = self.lastInitError {
-                throw NSError.init(domain: initError, code: 0, userInfo: nil)
+                throw NSError(domain: initError, code: 0, userInfo: nil)
             }
 
             // We check to see if we have a current logged in user. If we don't, then we need to sign someone in.
             // We throw an interactionRequired so that we trigger the interactive signin.
 
-            guard !(try authenticationProvider.users().isEmpty) else {
-                throw NSError.init(domain: "MSALErrorDomain",
-                                   code: MSALErrorCode.interactionRequired.rawValue,
+            // Acquire a token for an existing user silently
```

Comment:
Should we show here how to handle other errors (for example 'scopes declined' error)? See an example [here](https://github.com/microsoftgraph/ios-swift-snippets-sample/pull/13/files#diff-5d69b93abbd01a3e15b20579c104fbc4R24).